src/style.css

```diff
@@ -61,15 +61,20 @@ a:link:not(.btn):not(.unstyled):hover {
   border-bottom: 1px dotted #cc4141;
 }
 
-footer > a:not(.unstyled) {
-  @apply text-gray-300 !important;
+footer > div > a:not(.unstyled), footer > div > div > a:not(.unstyled)  {
+  @apply text-gray-300 !important
 }
 
-footer > a:not(.unstyled):hover {
+footer > div > a:not(.unstyled):hover, footer > div > div > a:not(.unstyled):hover {
```

Comment:
it looks like a pretty complicated selector, any chance we could do better ?

Reply:
Done